back-office/multistoregeneralspecs.md

```diff
@@ -0,0 +1,192 @@
+# SPECS GENERAL MULTISTORE BACK-OFFICE
+
+As a merchant I need to be able to correctly manage all my stores
+
+## WHAT
+
+Thus, this project aims at specifying the current and expected behaviour when the multistore is enabled and when there are many stores. It's the general behavior. Some pages can have specific behavior, you can find these specials cases in back-office/multistorespecialsspecs.md
+
+## SPECIFICATIONS
+
+### Shop & shop group color
+
+While creating a shop or a group, the user can select a color as described in the specs to setup the multistore<br/>
+This color will be displayed in the multistore header for the selected shop or shop group.<br/>
+It will be also displayed before the name of each shop / group in the context drop-down.
+
+### Multistore header
+
+On each pages, when the multistore feature is enabled and if many shops are created, a multistore header is displayed.<br/>
+The multistore header contains many things listed below:
+
+**Color**
+
+This header takes the color of the selected context:<br/>
+- The all shops context is blue and this color can't be changed by the user.<br/>
+- The shop group color can be configured while creating or editing the group<br/>
+- The shop color can be configured while creating or editing the shop.
+
+If the shop or the group selected doesn't have yet a color configured, the default color is grey.<br/>
+In this case, an information message is displayed on the page to prompt the user to define a color with a link opening the store's or group's page edition.<br/>
```

Comment:
@LouiseBonnard message to be defined

Reply:
I suggest _Customize your multistore header, follow the link to pick a color for this shop context:_ in Admin.Navigation.Header

Reply:
As the color can be for a shop or a group, wdyt of _Customize your multistore header, follow the link to pick a color for this context_ ?

Reply:
In the wording defined so far, _shop context_ stands for the context of the user. It means hat it can be:

- a single shop context
- a group of shops context
- an all shops context 

You can find another mention of _shop context_ in the color picker tooltip. I find it clear enough but if you find it too confusing, no prob, I can change the two wordings mentioning it. :-)

Reply:
All right, so let's keep it like this :+1: